.vscode/extensions.json

```diff
@@ -5,10 +5,10 @@
     "editorconfig.editorconfig",
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
-    "ms-ossdata.vscode-pgsql",
     "ms-python.python",
     "ms-python.vscode-pylance",
     "redhat.vscode-yaml",
+    "supabase.postgrestools",
```

Comment:
This LSP is better, and I think my original suggestion wasn't as useful as I thought.